elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "NoRedInk/noredink-ui",
     "summary": "UI Widgets we use at NRI",
     "license": "BSD-3-Clause",
-    "version": "12.20.0",
```

Comment:
would you mind reverting this? We don't bump versions until the change is on the main branch, just so we can bundle a bunch of stuff together without getting conflicting bumps

Reply:
Oh sorry! This was not supposed to be here!

Reply:
we're already on 12.21.0, which is odd!

Reply:
I've just removed this as it was an unintended change.
Thank you!